### PR TITLE
Relnotes 414 pa

### PR DESCRIPTION
**Pull request by PaulAngus**
improve information to external DNS note and reformat line widths.

### CODE REVIEW

**Comment by PaulAngus** [comment]
request docbuild

**Comment by PaulAngus** [comment]
request docbuild

File: source/releasenotes/about.rst [hunk 1]
```diff
@@ -17,7 +17,8 @@
 What's New in |release|
 =======================
 
-Apache CloudStack |release| is a |version| LTS release with over 15 major new features, and over 200 enhancements and fixes since 4.13.  Highlights include:
+Apache CloudStack |release| is a |version| LTS release with over 15 major new features, and
+over 200 enhancements and fixes since 4.13.  Highlights include:
```
Comment (andrijapanicsb) [inline source/releasenotes/about.rst:21]: Any line break/making those shorter in source docs will NOT affect the output - it's identical, but it doesn't hurt to make it beautiful.

Comment (PaulAngus) [reply source/releasenotes/about.rst:21]: I restructured the entire documentation set; I'm fully aware that a new line at c.80 characters does not make any difference to the final  output.  However, we have a 'standard', so realigned the text while adding some explanation.

**Pull request merged**